extension/src/setup/studio.ts

```diff
@@ -205,30 +210,47 @@ export class Studio extends Disposable {
     })
   }
 
+  private resetStudioValues(
+    previousStudioUrl: string,
+    previousStudioAccessToken: string | undefined
+  ) {
+    this.studioAccessToken = undefined
+    this.shareLiveToStudio = undefined
+    this.studioUrl = DEFAULT_STUDIO_URL
+
+    if (previousStudioAccessToken || previousStudioUrl !== DEFAULT_STUDIO_URL) {
+      this.studioConnectionChanged.fire()
+    }
+  }
+
   private async setStudioValues() {
     const cwd = this.getCwd()
 
     const previousStudioAccessToken = this.studioAccessToken
+    const previousStudioUrl = this.studioUrl
 
     if (!cwd) {
-      this.studioAccessToken = undefined
-      this.shareLiveToStudio = undefined
-
-      if (previousStudioAccessToken) {
-        this.studioConnectionChanged.fire()
-      }
+      this.resetStudioValues(previousStudioUrl, previousStudioAccessToken)
       return
     }
 
-    const [studioAccessToken, shareLiveToStudio] = await Promise.all([
-      this.accessConfig(cwd, ConfigKey.STUDIO_TOKEN),
-      (await this.accessConfig(cwd, ConfigKey.STUDIO_OFFLINE)) !== 'true'
-    ])
+    const [studioAccessToken, shareLiveToStudio, studioUrl] = await Promise.all(
+      [
+        this.accessConfig(cwd, ConfigKey.STUDIO_TOKEN),
+        (await this.accessConfig(cwd, ConfigKey.STUDIO_OFFLINE)) !== 'true',
+        (await this.accessConfig(cwd, ConfigKey.STUDIO_URL)) ||
```

Comment:
We default to `https://studio.iterative.ai` if the url is not set found in the config.

Reply:
[Q] Does `dvc config` return an empty string when there is no value present?

Reply:
Yes, it returns an empty string.